docs/gettingstarted/buildspecs_interface.rst

```diff
@@ -53,6 +53,20 @@ let's limit output to 5 entries, we can run the following
 
     .. command-output:: buildtest buildspec find --count=5
 
+The ``buildtest buildspec find`` command will fetch all buildspecs and display them in a table format, if you want to know how many records are displayed in the table you can use ``buildtest buildspec find --row-count`` which will display   
+a raw count of records in the table. In the example below we will run ``buildtest buildspec find --row-count`` which will show number of buildspecs from buildspec cache.
```

Comment:
I think we can rewrite this paragraph in less words to deliver same message. Take note you kinda repeat yourself three times referring to number of records 
 
1. how many records are displayed in the table
2. display   a raw count of records in the table
3. which will show number of buildspecs from buildspec cache.
___

You can use the `--row-count` option to retrieve total number of records from the **buildtest buildspec find** query. In the next example, we show retrieve number of tests  available in the buildspec cache by running the following.

Reply:
Right. Makes sense. I have made the changes.